### ISSUE TITLE
A quick question about implementation details

### ISSUE THREAD

**Issue by HelloWorldLTY** [PharMolix/LangCell#6]
Hi, thanks for your interesting work. I am a bit confused about two settings in your model and I am looking forward to your reply :)

1. I think according to the text description, you had text encoder and cell encoder participating for pre-training, and these two encoders will be used for zero-shot inference. However, in the zero-shot tutorial you provided, you still use the pubmed data to initialize the text encoder (not the version after your pre-training). Are there any intuitions behind this design?
<img width="418" alt="image" src="https://github.com/PharMolix/LangCell/assets/43333475/ad6d0b65-a7b8-4e30-8a55-ff9de3698cda">

2. In the ablation test folder, I can only find one result for L_CTM. Have your tried ablations for L_CC or L_CT? Thanks.

**Comment by toycat-I** [comment]
Thank you for your interest in LangCell! 
1. In fact, in the zero-shot tutorial, we use pre-trained checkpoints to initialize the text encoder. pubmedBERT is only used to initialize the tokenizer.
```
text_pretrained_model = 'microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract-fulltext'
tokenizer = BertTokenizer.from_pretrained(text_pretrained_model)
# ......
text_encoder = MedBertModel.from_pretrained('ckpt/text_bert', add_pooling_layer=True)
# ......

```

2. The specific ablation experimental setup can be found in the paper. Due to the huge computational overhead, it is impossible to try all combinations. We verified the contribution of each loss function in a cost-effective manner.
![image](https://github.com/PharMolix/LangCell/assets/126679141/fa9142e0-01ab-49af-9a6a-0de0f180f0bb)
